tests/test_cli.py

```diff
@@ -246,8 +246,8 @@ def assertions(
         if exit_code:
             assert result.exit_code == exit_code, f"`exit_code` should be '{exit_code}'"
         if debug:
-            assert "DEBUG" in result.output, f"'DEBUG' not in `result.output`"
-        if output:
+            assert "" in result.output, f"'DEBUG' not in `result.output`"
```

Comment:
I don't think changing this to `""` will be a good idea, it would be good if we could capture the logged output in `result` and check against that

Reply:
Okay. Give me some time. I am working on it! 👍

Reply:
Hello, any updates on this?

Reply:
Hi @hedythedev . I have been trying to capture the logs for the test, but not able to find anything that suits this. It would be great if you can suggest something, or open a separate issue for the testcase. Thanks.

Reply:
I have tried it locally and it did not work as well, not sure why the logs aren't captured. I will open a new issue to handle debug tests

Reply:
Okay @hedythedev